src/js/app.js

```diff
@@ -0,0 +1,139 @@
+import FilterPage from './containers/filter-page';
+import SettingsPage from './containers/settings-page';
+
+import AddPointPage from './containers/wizard/add-point-page';
+import UpdatePointPage from './containers/wizard/update-point-page';
+import AddAlertPage from './containers/wizard/add-alert-page';
+
+import PointLocation from './components/wizard/point-location';
+import ServiceName from './components/wizard/service-name';
+import ServiceDescription from './components/wizard/service-description';
+import ServiceHours from './components/wizard/service-hours';
+import ServiceAmenities from './components/wizard/service-amenities';
+import AlertNameDescription from './components/wizard/alert-name-description';
+
+import PeekPointCard from './components/point-card/peek-point-card';
+import ViewPointCard from './components/point-card/view-point-card';
+import RatingPointCard from './components/point-card/rating-point-card';
+
+// Exposes the history object for navigating with the React Router.
+// Right now, it's hash based.
+import history from './history';
+
+// Builds a redux store given optional middleware.
+// TODO: refactor out this dependency once we only depend on redux-thunk.
+import StoreBuilder from './store';
+
+// Allow direct access to the local PouchDB instance and import the gateway
+// singleton to make working with the database easier.
+import database from './database';
+import gateway from './gateway';
+
+// Sync and NetworkManager allow us to automatically work with the remote
+// PouchDB when the user is online.
+import Sync from './sync';
+import { NetworkManager } from './reducers/network';
+
+// Redux action creators used by app.js
+import { reloadPoints } from './reducers/points';
+
+// Load global css used by any other js module.
+import '../../node_modules/leaflet/dist/leaflet.css';
+import '../css/app.css';
+
+// Fix tap events so material-ui components work
+import injectTapEventPlugin from 'react-tap-event-plugin';
+injectTapEventPlugin();
+
+
+
+// When Cordova has loaded, assemble the application components and start them.
+//
+//  1. Create the redux store
+//  2. Start the NetworkManager to reflect network state in the store
+//  3. Start syncing the local PouchDB with the remote CouchDB
+//  4. Pull all locally available points into the store
+//  5. Start the React Router
+//
+// In order for this event to be fired, cordova.js must already be loaded via
+// a <script> tag. Also, a <div class="main" /> is required in the body. We
+// render the application into that div.
+document.addEventListener( 'deviceReady', ( ) => {
+  const storeBuilder = new StoreBuilder( [ gateway.getMiddleware() ] );
+  const store = storeBuilder.build();
+
+  const network = new NetworkManager( store );
+  network.monitor();
+
+  const sync = new Sync( database, gateway, store );
+  sync.start();
+
+  gateway.getPoints().then( points => {
+    store.dispatch( reloadPoints( points ) );
+  } );
+
+/*esfmt-ignore-start*/
```

Comment:
This is the largest section of JSX in the entire project. It's cleaner here to have the path and component attributes on the same line. Otherwise, esformatter puts all attributes on their own line. Here, it makes it hard to connect the component with the path.